src/perftest_parameters.h

```diff
@@ -178,8 +178,8 @@
 #define UC_MAX_RX     (16000)
 #define MIN_CQ_MOD    (1)
 #define MAX_CQ_MOD    (1024)
-#define MAX_INLINE    (912)
-#define MAX_INLINE_UD (884)
+#define MAX_INLINE    (1024)
+#define MAX_INLINE_UD (1024)
```

Comment:
This change can cause problems with devices with smaller max inline size.
the ibv_create_qp() will return NULL value if the MAX_INLINE isn't supported.

Reply:
At present, these two macros are only used for parameter verification to ensure that the inline length specified by the user through "- I" does not exceed this value.  By default, such a large inline size is not directly used. By the way, the current value are not supported for some HWs, but it works well.

In the current rdma core, if the driver does not support the value set by the user, the user will be informed of the max inline size that the driver can support in the returned attr.cap.max_inline_data. There is a example：https://github.com/linux-rdma/perftest/pull/178/commits/99decb7a2d1b1b12f985b04a9fa64f1ae25dfae3

Reply:
> This change can cause problems with devices with smaller max inline size. the ibv_create_qp() will return NULL value if the MAX_INLINE isn't supported.

Hi @sshaulnv 
As described in the link, the ibv_create_qp_ex returns the actual QP value via qp_init_attr_ex.
https://github.com/linux-rdma/rdma-core/blob/master/libibverbs/man/ibv_create_qp_ex.3

So, the value returned by the driver (in the qp_init_attr_ex.cap) is more suitable for determining whether the packet can be sent in inline mode.